### DISCUSSION TITLE
Release-timeline? Especially in regard of the next one (1.7)?

### DISCUSSION THREAD

**Discussion by marcelpetrick** [rauc/rauc#909] [General]
Hello,
I've see that the current development contains also the ability to encrypt rauc-bundles.  
Since this would be a needed feature, I started to search for something like a release-pipeline or when the next release is planned. But was not successful. Guessing based on the recent releases also gave a timeframe from "now" to "in two month".

Can you shed some light please?

**Comment by jluebbe** [comment]
Soon. :) There are no more features planned for v1.7.

We're currently testing current master in some of our projects. If you could try it as well and report whether it works, it would help with making sure that it's ready for release.

**Reply by marcelpetrick** [reply]
Ticket can be closed with the release of the 1.7 on 3rd June. Thank you.
